src/bb-library/Server/Manager/PleskApiClient.php

```diff
@@ -0,0 +1,1034 @@
+<?php
+/**
+ * BoxBilling
+ *
+ * LICENSE
+00 *
```

Comment:
this will cause error

Reply:
@fordnox I have updated the comment block to remove the ambiguous characters.